### PR TITLE
Fix some minor issues in documentation

### PR DESCRIPTION
**Pull request by btoll**
https://www.grammar-monster.com/easily_confused/its_its.htm
https://www.grammarly.com/blog/whos-whose/

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ jobs:
     - Echo
 ```
 
-Here we have a mule config file that lists one task and one job. The tasks field is where you document task configurations that you will need to execute your jobs. It takes a list of maps (dicts in python) that are used to initialize your tasks. One key thing to keep in mind here is that every task configuration must have a task field set. This is used by mule to loop up the task you wish to be initialize. Every other field is a configuration for your task. In this example we have one task called Echo, which simply prints out "Message for the console".
+Here we have a mule config file that lists one task and one job. The tasks field is where you document task configurations that you will need to execute your jobs. It takes a list of maps (dicts in python) that are used to initialize your tasks. One key thing to keep in mind here is that every task configuration must have a task field set. This is used by mule to loop through the task you wish to be initialize. Every other field is a configuration for your task. In this example, we have one task called Echo, which simply prints out "Message for the console".
```
Comment (btoll) [inline docs/getting_started.md]: Here I changed "loop up" to "loop through".

Comment (onetechnical) [inline docs/getting_started.md]:
```suggestion
Here we have a mule config file that lists one task and one job. The tasks field is where you document task configurations that you will need to execute your jobs. It takes a list of maps (dicts in python) that are used to initialize your tasks. One key thing to keep in mind here is that every task configuration must have a task field set. This is used by mule to add the task you wish to be initialized. Every other field is a configuration for your task. In this example, we have one task called Echo, which simply prints out "Message for the console".
```

**Review by bricerisingalgorand — DISMISSED** [review]
I approve of this message

File: docs/getting_started.md (hunk 1 above)
Comment (onetechnical) [reply docs/getting_started.md]: This sentence is awkward...I think the idea was to add the task (hence the above suggestion). Regardless, the "initialize" needs to be modified, whichever way it goes.

**Review by onetechnical — CHANGES_REQUESTED** [review]
Ick, a grammar PR! Grammared the grammar changes.

**Review by onetechnical — APPROVED** [review]

**Review by bricerisingalgorand — APPROVED** [review]
I still approve of this message

**Pull request merged**